### ISSUE TITLE
Add more data to the leaderboards, like rating and win percentage

### ISSUE THREAD

**Issue by Marc-Spector** [FAForever/downlords-faf-client#2526]
From forum https://forum.faforever.com/topic/3206/small-suggestions-topic/40?_=1642362790786

**Comment by Sheikah45** [comment]
It was an active choice not to display rating @BlackYps  could say more

**Comment by BlackYps** [comment]
Yes, the point of the league system is to hide the rating. Adding a winrate is a good idea though

**Comment by BlackYps** [comment]
We need the league service to tell us the number of won and lost games for this
